docs/variant_calling.md

```diff
@@ -0,0 +1,57 @@
+# Variant calling <!-- omit in toc -->
+
+- [Germline variant calling](#germline-variant-calling)
+- [Somatic variant calling with tumor - normal pairs](#somatic-variant-calling-with-tumor---normal-pairs)
+- [Somatic variant calling with tumor only samples](#somatic-variant-calling-with-tumor-only-samples)
+
+## Germline variant calling
+
+Using Sarek, germline variant calling will be performed always, if a variant calling tool is selected and it allows for germline variant calling.
```

Comment:
```suggestion
Using Sarek, germline variant calling will always be performed if a variant calling tool is selected, and it germline variant calling is turned on.
```

Reply:
No sure about the phrasing for the last part.
```suggestion
Using Sarek, germline variant calling will always be performed if a variant calling tool with a germline mode is selected.
```